src/components/UsageBar.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import UsageBarItem from "./UsageBarItem";
+import {
+  RANK_COMMANDER,
+  RANK_OPERATIVE,
+  RANK_CORPS,
+  RANK_SPECIAL_FORCES,
+  RANK_HEAVY,
+  RANK_SUPPORT
+} from "../util/constants";
+
+const UsageBarBody = styled.div`
+  grid-column: 1 / span 2;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom: 0;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
+const UsageBar = ({
+  points,
+  commanderSlots,
+  operativeSlots,
+  corpsSlots,
+  specialForcesSlots,
+  supportSlots,
+  heavySlots
```

Comment:
Optional:
Are these actually different? Maybe should this just be an object with `RANK_COMMANDER` etc as keys and the slots as the value? It's pretty expressive which is nice, but you could save some lines if you wanted to by iterating over an object instead.

Reply:
Had a similar though, this way may be annoying to update as well

Reply:
I was thinking the same but I'm kind of on the fence. Given that this list is basically static, there is value in having everything explicitly stated and clearly visible. You do lose a bit of clarity by just iterating through the list for render. 

Suppose the main thing is how much this is expected to change over time.

if (changeALot || unknownAmountOfChangeButKindOfLikelyTo) {
    create object and iterate, passing the props to the usage bar item. Update config object with new properties and update the single place where UsageBarItem is referenced.
} else {
    Keep these explicit for clarity and in the rare case it changes it's like, 5 minutes to update each of these
}

Reply:
Agreed 👌

Reply:
most actions a user takes will affect at least one of these, pints being the most common

Reply:
Ok, came up with something 👍